### DISCUSSION TITLE
How do I delete models in the new version?

### DISCUSSION THREAD

**Discussion by qlkv** [open-webui/open-webui#7618] [Q&A] [answered]
The new version of the model management UI (v0.4.7) is extremely unusable, I can't find how to delete a model, I can't find how to add a new model. The previous version was a much better experience. 

Any hints how to add a new model or remove existing one?

**Comment by flefevre** [comment, marked as answer]
You have to go to admin panel, then models, then ollama conneciton and management
see below


![del](https://github.com/user-attachments/assets/4a9ecfe3-b20f-46af-90c5-629ffc7430e3)

![del2](https://github.com/user-attachments/assets/e14511bd-1c70-4624-9a88-ac5f20d2e98f)
